Build of docker image eclipse/che-mount:nightly failed

### Description
Building of Docker Image eclipse/che-mount:nightly is failing because next command:
> /bin/sh -c apk add --update build-base curl bash sshfs &&     apk add ocaml --update-cache --repository http://dl-3.alpinelinux.org/alpine/edge/community/ --allow-untrusted &&     curl -L https://www.seas.upenn.edu/~bcpierce/unison/download/releases/unison-$UNISON_VERSION/unison-$UNISON_VERSION.tar.gz | tar xzv -C /tmp &&     cd /tmp/src &&     sed -i -e 's/GLIBC_SUPPORT_INOTIFY 0/GLIBC_SUPPORT_INOTIFY 1/' fsmonitor/linux/inotify_stubs.c &&     make &&     cp /tmp/src/unison /usr/local/bin &&     cp /tmp/src/unison-fsmonitor /usr/local/bin &&     apk del ocaml curl build-base bash &&     rm -rf /tmp /var/cache/apk/* &&     mkdir /mntssh && chmod -R 777 /mntssh &&     mkdir /mnthost && chmod -R 777 /mnthost

returned a non-zero code: 2
```
File "/tmp/src/unicode.ml", line 25, characters 16-33:
Warning 3: deprecated: String.unsafe_set
File "/tmp/src/unicode.ml", line 822, characters 20-21:
Error: This expression has type string but an expression was expected of type
         bytes
make: *** [unicode.cmx] Error 2
Makefile.OCaml:434: recipe for target 'unicode.cmx' failed
```

This error blocks nightly build of Eclipse Che images here https://ci.codenvycorp.com/view/qa/job/all-che-docker-images-nightly

--- a/dockerfiles/mount/Dockerfile
+++ b/dockerfiles/mount/Dockerfile
@@ -38,7 +38,7 @@
 #   /bin/synch.sh <ip> <ws-port>
 FROM alpine:3.4
 
-ENV UNISON_VERSION=2.48.4
+ENV UNISON_VERSION=2.51.2
 
 RUN apk add --update build-base curl bash sshfs && \
     apk add ocaml --update-cache --repository http://dl-3.alpinelinux.org/alpine/edge/community/ --allow-untrusted && \

